assets/js/googlesitekit/widgets/datastore/areas.test.js

```diff
@@ -493,6 +494,103 @@ describe( 'core/widgets Widget areas', () => {
 			} );
 		} );
 
+		describe( 'isWidgetAreaActive', () => {
+			beforeEach( () => {
+				registry
+					.dispatch( CORE_WIDGETS )
+					.registerWidgetArea( 'TestArea', {
+						title: 'Test Header',
+						subtitle: 'Cool stuff for yoursite.com',
+						style: 'composite',
+					} );
+			} );
+
+			it( 'requires a widgetAreaSlug', () => {
+				expect( () => {
+					registry.select( CORE_WIDGETS ).isWidgetAreaActive();
+				} ).toThrow(
+					'widgetAreaSlug is required to check a widget area is active.'
+				);
+			} );
+
+			it( 'returns false if there are no widgets registered for the area', () => {
+				expect(
+					registry
+						.select( CORE_WIDGETS )
+						.isWidgetAreaActive( 'TestArea' )
+				).toEqual( false );
+			} );
+
+			describe( 'when there are widgets registered for the area', () => {
```

Comment:
Once again this is a bit over-engineered. Could you please hoist tests defined in this `describe` block to the upper level. Just define two widget areas (one with widgets and one empty) to test both conditions.

Reply:
No problem, that's done :+1: